test/extension/handlers/topologymutation/handler_test.go

```diff
@@ -274,6 +275,76 @@ func Test_patchKubeadmConfigTemplate(t *testing.T) {
 			},
 			expectedTemplate: &bootstrapv1.KubeadmConfigTemplate{},
 		},
+		{
+			name:             "fails if builtin.machinePool.class is not set",
```

Comment:
I think this is the same test case as in l.209. Let's drop this one here and change the first one to "fails if builtin variable is not set"